nav2_smoother/src/simple_smoother.cpp

```diff
@@ -81,29 +81,27 @@ bool SimpleSmoother::smooth(
   std::lock_guard<nav2_costmap_2d::Costmap2D::mutex_t> lock(*(costmap->getMutex()));
 
   for (unsigned int i = 0; i != path_segments.size(); i++) {
-    if (path_segments[i].end - path_segments[i].start > 9) {
```

Comment:
The path segments still need to be large enough to smooth, i.e. at least size 3, if not that size, it cannot be smoothed any more than it is. I think you just need to update 9 to 3 for this case. It was 9 due to copy+pasting from the SG Filter Smoother in the other plugin which does require length > 9 due to its formulation

Reply:
If we have a path of 1, we can't smooth it. If we have a path of 2, there's nothing to smooth, its always a straight line. Anything less than 3 is meaningless to 'smooth' because its a null operation.